pkgdown/_pkgdown.yml

```diff
@@ -104,6 +104,14 @@ reference:
   - dm_unnest_tbl
   - dm_unpack_tbl
 
+- title: Data frames operations
```

Comment:
Operations on data frames and lazy tables

Reply:
done in https://github.com/cynkra/dm/pull/918/commits/d5a30be53c3975166e9c69e2480f532a314ef29f